test/e2e/upgrade_management_cluster_test.go

```diff
@@ -235,6 +236,7 @@ func preCleanupManagementCluster(clusterProxy framework.ClusterProxy) {
 			By("Install Ironic in the source cluster as containers")
 			bmoPath := e2eConfig.GetVariable("BMOPATH")
 			ironicCommand := bmoPath + "/tools/run_local_ironic.sh"
+			//#nosec G204 -- This is just a test. We are not concerned with command injection
```

Comment:
Same here, even though this one is a bit "simpler". We could just hard code the BMOPATH for now but since we have a variable for it I would prefer not to. I think we can rework it in a better way but it will require larger changes that I would prefer to do as a separate PR.

Reply:
Its not a new bug, so just create a follow-up ticket to get it resolved later.